d2bs/kolbot/libs/bots/MFHelper.js

```diff
@@ -169,16 +169,22 @@ MainLoop:
 						this.clearCowLevel();
 						delay(1000);
 
-						if (!Pather.usePortal(null, player.name)) {
+						if (!Pather.getPortal(null, player.name)) {
```

Comment:
This can be a single if condition:
if (!Pather.getPortal(...) || !Pather.usePortal(...))

Reply:
I don't know why usePortal is slow, but from experience I know that the way it works currently when MFHelper misses a portal to town (the one the leader made when they left), it takes a noticeable pause before making their own portal. If we use getPortal before using usePortal, there is effectively no delay. What you recommend would work well too, I think.